WooCommerce/Classes/POS/Presentation/Item Selector/CouponCardView.swift

```diff
@@ -17,38 +17,97 @@ struct CouponCardView: View {
 
     var body: some View {
         HStack(spacing: Constants.cardSpacing) {
-            POSCouponImageView(size: dimension)
+            POSCouponImageView(size: dimension,
+                               state: coupon.isExpired ? .expired : .normal)
 
             VStack(alignment: .leading, spacing: Constants.textSpacing) {
                 Text(coupon.code)
-                    .foregroundStyle(Constants.titleColor)
+                    .foregroundStyle(titleColor)
                     .multilineTextAlignment(.leading)
                     .font(Constants.itemTitleFont)
 
                 Text(coupon.summary)
-                    .foregroundStyle(Constants.detailColor)
+                    .foregroundStyle(summaryColor)
                     .font(Constants.itemDetailFont)
                     .lineLimit(2)
                     .fixedSize(horizontal: false, vertical: true)
+
+                if coupon.isExpired, let expirationDate = coupon.dateExpires {
+                    Text(formattedExpirationText(date: expirationDate))
+                        .foregroundStyle(expirationTextColor)
+                        .font(Constants.itemDetailFont)
+                        .lineLimit(1)
+                }
             }
             .padding(.horizontal, Constants.horizontalTextPadding * (1 / scale))
             .padding(.vertical, Constants.verticalTextPadding * (1 / scale))
             Spacer()
         }
         .frame(maxWidth: .infinity, minHeight: dimension)
-        .background(Constants.backgroundColor)
+        .background(cardBackgroundColor)
         .posItemCardBorderStyles()
     }
 }
 
 private extension CouponCardView {
     typealias Constants = PointOfSaleItemListCardConstants
+
+    enum Localization {
+        static let expirationText = NSLocalizedString(
+            "couponCardView.expirationText",
+            value: "Expired · %@",
```

Comment:
Reminder to do before the merge 👍 
```suggestion
            value: "Expired on %@",
```